README.md

```
@@ -176,3 +176,36 @@ contribute and how to run the unit tests and style checks locally.
 By participating in this project and its community, you are expected to uphold
 this code.
+
+
+## Docker deployment
+
+The project includes a Docker setup to run phpList with Apache and MySQL.
+
+Quick start (development/test):
+
+- Build and start the stack: `docker compose up --build`
+- Open http://localhost:8080
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)